MirandaRoland/StoreProject/src/Package.java

```diff
@@ -0,0 +1,262 @@
+import java.awt.HeadlessException;
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.Scanner;
+import javax.swing.JOptionPane;
+
+/**
+ this class will encapsulate a package of the objects stored in the store;
+ */
+public class Package extends GUI implements Comparable,Serializable, displayable {
+    private String code;
+    private long amount=0;
+    private String name;
+    private String discription;
+    private double price;
+    private long dispatched=0;
+    private long issued=0;
+
+    public Package(String code, String name, long amount, double price, String discription) {
+        this.code = code;
+        this.amount = amount;
+        this.name = name;
+        this.discription = discription;
+        this.price = price;
+    }
+    public Package(){
+        this(null,null,0,0,null);
```

Comment:
You should not call the other constructor objects by default are initialized to null